src/common/components/landing-cta/LandingCta.tsx

```diff
@@ -9,24 +9,24 @@ const LandingCta = () => {
   const { t } = useTranslation('landing');
 
   return (
-    <div className="grid md:grid-cols-2 grid-cols-1">
-      <LandingCtaItem
-        backgroundColor="bg-yellow-500"
-        title={t('cta.title_1')}
-        description={t('cta.description_1')}
-        buttonText={t('cta.register')}
-        buttonClasses="yellow-button bg-black text-white hover:bg-black-800"
-        onClick={() => openInNewTab(process.env.REACT_APP_CREATE_ONG_PROFILE_LINK!)}
-      />
-      <LandingCtaItem
-        backgroundColor="bg-gray-100"
-        title={t('cta.title_2')}
-        description={t('cta.description_2')}
-        buttonText={t('cta.discover_ongs')}
-        buttonClasses="yellow-button "
-        onClick={() => navigate('/organizations')}
-        anchorLinkText={t('cta.p4g_link')}
-      />
+    <div className="w-full md:right-split-color bottom-split-color">
+      <div className="wrapper grid md:grid-cols-2 grid-cols-1 lg:gap-x-56 gap-x-40 gap-y-24">
+        <LandingCtaItem
```

Comment:
La fel si aici, wrapperele putem sa le tinem in afara, altfel componentele nu sunt foarte reutilizabile